modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -349,9 +336,6 @@ export default {
           draft: true
         }
       ));
-      if (!this.mounted) {
```

Comment:
Continuing here will do no harm.

Reply:
moduleOptions is immutable, so should be harmless to call here even if unmounted.